ssas/service/main/main.go

```diff
@@ -337,3 +358,36 @@ func closeRows(rows *sql.Rows) {
 func cliTrackingID() string {
 	return fmt.Sprintf("cli-command-%d", time.Now().Unix())
 }
+
+func showXData(clientID, auth string) error {
+	// The auth header decoding logic was pulled from Go's requuest.go#parseBasicAuth func
+	if auth != "" {
+		c, err := base64.StdEncoding.DecodeString(auth)
+		if err != nil {
+			return errors.New("unable to decode the auth hash: " + err.Error())
```

Comment:
You can also use `fmt.Errorf` so you don't have to do all of this string concatenation.

So this could be:
```
return fmt.Errorf("unable to decode the auth hash: %w", err)
```

The `%w` allows you to leverage Go's error wrapping/chaining

Ref: https://blog.golang.org/go1.13-errors